src/test/java/io/github/eocqrs/kafka/fake/FkConsumerTest.java

```diff
@@ -241,6 +242,87 @@ public void onPartitionsAssigned(final Collection<TopicPartition> collection) {
     );
   }
 
+  @Test
+  void unsubscribes() throws Exception {
+    final String topic = "unsubscribe.test";
+    final UUID uuid = UUID.randomUUID();
+    final Consumer<String, String> consumer =
+      new FkConsumer<>(
+        uuid,
+        this.broker
+      );
```

Comment:
```suggestion

  @Test
  @ExtendWith({
    UUIDExtension.class,
    ConsumerExtension.class 
  })
  void unsubscribes(final UUID uuid, final Consumer<String, String> consumer) throws Exception {
    final String topic = "unsubscribe.test";
```

Reply:
@l3r8yJ Unfortunately it is not possible to mix the params using extensions

if we need a UUID extension and Consumer Extension

when creating a consumer, we need a UUID from the extension, that generates it
but I can't take this generated parameter

Then, if we decide to make only one `ConsumerExtension`, in following assertion we don't have parameter uuid:

```java
MatcherAssert.assertThat(
      "Consumer ID in subscription is not present",
      this.broker.data(
          "broker/subs/sub[consumer = '%s']/consumer/text()"
            .formatted(
              uuid
            )
        )
        .isEmpty(),
      Matchers.equalTo(true)
    );
```

maybe I missing something, especially in getting parameters from another extension

WDYT?

Reply:
@l3r8yJ also, this test case (unsubscribes) requires testing consumers with the same broker